x/bank/internal/keeper/app_test.go

```diff
@@ -1,11 +1,13 @@
-package bank
+package keeper_test
```

Comment:
not sure if the file name is accurate here, seems like it's a combination of the `test_common` and msg tests

Reply:
it's a test package, required to avoid circular dependencies

Reply:
...that could actually be avoided once this moved into types